WalletWasabi.Fluent/Models/Wallets/WalletRepository.cs

```diff
@@ -115,7 +113,21 @@ private async Task<IWalletSettingsModel> CreateNewWalletAsync(WalletCreationOpti
 					{
 						TipHeight = Services.SmartHeaderChain.TipHeight
 					};
-					return walletGenerator.GenerateWallet(walletName, password, mnemonic);
+
+					return walletBackup switch
+					{
+						RecoveryWordsBackup recoveryWordsBackup =>
+							walletGenerator.GenerateWallet(
+								walletName,
+								recoveryWordsBackup.Password!,
+								recoveryWordsBackup.Mnemonic).Item1,
+						MultiShareBackup multiShareBackup =>
+							walletGenerator.GenerateWallet(
+								walletName,
+								multiShareBackup.Password!,
+								multiShareBackup.Shares?.Take(multiShareBackup.Settings.Threshold).ToArray()).Item1,
```

Comment:
Add names to the tuple, please.

Reply:
https://github.com/WalletWasabi/WalletWasabi/pull/13800/commits/3034f4f480d865a22fd216f6bfebcae0559297b5